dist/installer.php

```diff
@@ -108,6 +118,21 @@ public function run() {
 
         $this->output($this->cliName . " installer", 'heading');
 
+        if ($this->migratePrompt && !getenv($this->envPrefix . 'WRAPPED') && !$this->isCI()) {
```

Comment:
Is there any chance this runs with `PLATFORM_WRAPPED=1`?

Reply:
Good point, no